packages/fast-form-generator-react/src/form/form-item.select.style.ts

```diff
@@ -0,0 +1,70 @@
+import { Direction, ellipsis, localizeSpacing, toPx } from "@microsoft/fast-jss-utilities";
+
+function applypaddingStyle(): any {
+    return {
+        padding: localizeSpacing(Direction.ltr)(`${toPx(8)} ${toPx(24)} ${toPx(8)} ${toPx(8)}`)
+    };
+}
+
+const styles: any = {
```

Comment:
this should not be any, see fast-components-styles-msft/src/checkbox/index.ts for example, should be something like `ComponentStyles<ISelectClassNameContract, IDesignSystem>`

Reply:
Good catch - updating